queue_classic_plus.gemspec

```diff
@@ -19,6 +19,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "queue_classic", ">= 3.1.0"
-  spec.add_development_dependency "bundler", "~> 1.6"
+  if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.3.0')
```

Comment:
Is 2.3.0 too aggressive to start using bundler 2, wasn't it only recently introduced? What happens if we just leave it pegged to 1.6?

Reply:
This `if` is only because Bundler 2 doesn't support Ruby < 2.3.0.

Bundler 2 is from January, so ~7 months now with no real issues. Changelog:
> The following is the full list of changes to Bundler 2:
>
>    Removed support for Ruby < 2.3
>    Removed support for RubyGems < 3.0.0
>    Changed the `github: 'some/repo'` gem source to use the `https` schema by default
>    Errors/warnings will now print to `STDERR`
>    Bundler now auto-switches between version 1 and 2 based on the Lockfile

We could probably drop support for Ruby < 2.3 here as well? 🤷‍♂ 

> What happens if we just leave it pegged to 1.6?

It's just super annoying to work with locally.